### DISCUSSION TITLE
Create role for OAUTH2

### DISCUSSION THREAD

**Discussion by code23rus** [apache/superset#17507] [Q&A / Help] [answered]
Hi! 
I have configured OAuth2 authorization with my server, and automatic user registration (AUTH_USER_REGISTRATION = True). But all users are created with the role specified in the AUTH_USER_REGISTRATION_ROLE parameter. 
How can I make some users create with a different role, in particular "Admin"?

**Comment by kamalkeshavani-aiinside** [comment, marked as answer]
You can use `AUTH_USER_REGISTRATION_ROLE_JMESPATH` config. More info here:
https://flask-appbuilder.readthedocs.io/en/latest/config.html#using-jmespath-to-map-user-registration-role

**Reply by code23rus** [reply]
It's worked, thank you!
